BUILD.bazel

```diff
@@ -0,0 +1,6 @@
+config_setting(
+    name = "index_while_building_v2",
+    values = {
+        "features": "swift.index_while_building_v2",
```

Comment:
load the feature name from rules_swift instead of hard-coding the string?

Reply:
👍  hope we can do this, I've gotta look to see if those symbols are exported